optimum/utils/normalized_config.py

```diff
@@ -254,6 +254,10 @@ class NormalizedConfigManager:
         "xlm-roberta": NormalizedTextConfig,
         "yolos": NormalizedVisionConfig,
         "mpt": MPTNormalizedTextConfig,
+        "mixformer-sequential": GPTBigCodeNormalizedTextConfig,
+        "baichuan": NormalizedTextConfig,
+        "qwen": NormalizedTextConfig,
+        "chatglm": NormalizedTextConfig.with_args(num_layers="num_layers"),
```

Comment:
what about the vocab size, shouldn't it be `padded_vocab_size` ?
 https://huggingface.co/THUDM/chatglm3-6b/blob/main/config.json#L32

Reply:
chatglm has 3 models , I find chatglm is `vocab_size`, but chatglm2&3 is `padded_vocab_size`，Could you help me deal with this situation? @echarlaix 
https://huggingface.co/THUDM/chatglm-6b/blob/main/config.json#L27